hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -319,18 +331,18 @@ private static Boolean tableExists(Connection conn, Map<String, String> options)
    * @return
    * @throws Exception
    */
-  public static Schema getJDBCSchema(Map<String, String> options) throws Exception {
-    Connection conn = createConnectionFactory(options);
-    String url = options.get(JDBCOptions.JDBC_URL());
-    String table = options.get(JDBCOptions.JDBC_TABLE_NAME());
-    boolean tableExists = tableExists(conn,options);
+  public static Schema getJDBCSchema(final Map<String, String> options) throws Exception {
+    final Connection conn = createConnectionFactory(options);
```

Comment:
I get where you are coming from for `final` but given we are not following this everywhere.. can we please limit this PR to the minimal changes needed for this functionality..